modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_buttons.scss

```diff
@@ -17,4 +17,14 @@
     @include modular-spacing('margin-bottom', 0);
   }
 
+  // If they're in the same paragraph, get them to stack.
+  a[class^="su-button"] {
+    display: block;
+    width: fit-content;
```

Comment:
This property has IE11 issues. Have you looked to see what happens in that browser? We don't need it to be pixel perfect, just usable.

Reply:
lol, I had intended "max-content" but it still has issues with IE11

![Screen Shot 2020-04-09 at 6 12 20 PM](https://user-images.githubusercontent.com/284440/78953867-15a0e780-7a8f-11ea-9ab2-31e28bf5046b.png)
It is usable.